src/System.Net.Sockets/src/System/Net/Sockets/UDPClient.cs

```diff
@@ -233,11 +233,7 @@ protected virtual void Dispose(bool disposing)
         {
             if (disposing)
             {
-                if (GlobalLog.IsEnabled)
-                {
-                    GlobalLog.Print("UdpClient::Dispose()");
-                }
-
+                NetEventSource.Info(this);
```

Comment:
Makes sense.  I probably wouldn't want to do this in new code, but I understand that's beyond the scope here.